index.html

```diff
@@ -20,6 +17,11 @@
         <div class="header-inner">
           <nav class="header-nav">
             <a href="/" class="header-logo">Welcome</a>
+            <ul class="nav__list">
+              <li class="nav__item">Portoflio</li>
+              <li class="nav__item">About</li>
+              <li class="nav__item">Contact</li>
```

Comment:
- [ ] Please consider using an `<a>` tag for your navigation buttons 🦋, that way users can easily navigate through your page with a click of a button 📎. **_You will end up with something like this_** 👇🏽

![image](https://user-images.githubusercontent.com/43172164/200037646-92c66d03-e047-4a3f-9b91-097c3a0f9d75.png)
**_Please don't forget to style the anchor tags after adding so you stick to the design as much as possible_**